Subdivision_method_3/doc/Subdivision_method_3/Subdivision_method_3.txt

```diff
@@ -203,8 +197,7 @@ new points by cooperating with the `mask`.
 To implement Catmull-Clark subdivision,
 `Mask`, the <I>geometry policy</I>, has to realize the
 geometry masks of Catmull-Clark subdivision.
-The parameter `depth` specifies the iterations of the refinement
-on the control mesh.
+The number of iterations as well as the vertex point map can be specified using the named parameter `np`.
```

Comment:
I'll fix that later once https://github.com/CGAL/cgal/pull/2649 is through.

Reply:
What do you mean?

Reply:
I mean detailing a bit what named parameters are and linking to the NamedParameters.txt of the package.